src/assets/database.sql

```diff
@@ -406,7 +426,7 @@ INSERT INTO `tools` (`id`, `name`, `fullname`, `categories`, `description`, `sit
 (10, 'Sqlninja', 'Sqlninja', 'vulnerability-analysis  web-applications', 'https://tools.kali.org/vulnerability-analysis/sqlninja', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 'Vulnerability Analysis', 'Web Applications', NULL),
 (11, 'Powerfuzzer', 'Powerfuzzer', 'vulnerability-analysis  web-applications', 'https://tools.kali.org/vulnerability-analysis/powerfuzzer', 'https://www.powerfuzzer.com/', NULL, NULL, NULL, NULL, NULL, NULL, 'Vulnerability Analysis', 'Web Applications', NULL),
 (12, 'ProxyStrike', 'ProxyStrike', 'web-applications', 'https://tools.kali.org/web-applications/proxystrike', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 'Web Applications', NULL, NULL),
-(14, 'Dnsenum', 'Dnsenum', 'information-gathering', 'https://tools.kali.org/information-gathering/dnsenum', NULL, NULL, NULL, NULL, NULL, NULL, NULL, 'Information Gathering', NULL, NULL),
+(14, 'Dnsenum', 'Dnsenum', 'information-gathering','The main purpose of Dnsenum is to gather as much information as possible about a domain.', 'https://tools.kali.org/information-gathering/dnsenum', 'https://github.com/fwaeytens/dnsenum', 'Yes', NULL, 'dnsenum', NULL, NULL, 'Vulnerability Analysis', 'Information Gathering', NULL),
```

Comment:
The version installed through Dockerfile is the dnsenum2, so the corresponding github repository is the following: https://github.com/SparrowOchon/dnsenum2